packages/client/src/game/canvas/index.tsx

```diff
@@ -0,0 +1,284 @@
+import image1 from 'client/src/assets/images/game/1.png';
+import image2 from 'client/src/assets/images/game/2.png';
+import image3 from 'client/src/assets/images/game/3.png';
+import image4 from 'client/src/assets/images/game/4.png';
+import imageMoon from 'client/src/assets/images/game/moon.png';
+import imageBurn from 'client/src/assets/images/game/burn.png';
+import imageDestroyed from 'client/src/assets/images/game/destroyed.png';
+import imageMiss from 'client/src/assets/images/game/miss.png';
+import {CellStatus} from 'client/src/game/typing';
+import styles from 'client/src/pages/Game/PageSetShips/pageSetShips.module.scss';
+import React, {useState} from 'react';
```

Comment:
Может быть лучше так? И далее по коду тоже убрать `React`.
```suggestion
import {useState, useEffect, useRef} from 'react';
```

Reply:
Да, так лучше, сделаю. Там что-то осталось от Евгении, возможно там React.* использовался.